.gitignore

```diff
@@ -1,5 +1,4 @@
 node_modules/
-package-lock.json
```

Comment:
please add this back

Reply:
Sorry looks like that happened when I rebased it off my Mineflayer bump PR a week ago; fixed